Rakefile

```diff
@@ -624,29 +596,28 @@ namespace :kokoro do
         header "Using Ruby - #{RUBY_VERSION}"
         Rake::Task["kokoro:windows_acceptance_fix"].invoke
         sh "bundle update"
-        exit run_command_with_timeout("bundle exec rake ci:acceptance", 3600)
+        exit [
+          run_command_with_timeout("bundle exec rake ci:acceptance", 3600),
+          verify_in_gemfile(ENV["PACKAGE"])
+        ].max
       end
     end
   end
 
   task :release do
+    version = "0.1.0"
     header_2 ENV["JOB_TYPE"]
     Dir.chdir ENV["PACKAGE"] do
       Bundler.with_clean_env do
-        Rake::Task["kokoro:load_env_vars"].invoke
         header "Using Ruby - #{RUBY_VERSION}"
         sh "bundle update"
-        sh "bundle exec rake build"
-        gem = Dir.entries("pkg").select { |entry| File.file? "pkg/#{entry}" }.first
-        path = FileUtils.mkdir_p(File.expand_path("~") + "/.gem")
-        File.open("#{path.first}/credentials", "w") do |f|
-          f.puts "---"
-          f.puts ":rubygems_api_key: #{ENV["RUBYGEMS_API_TOKEN"]}"
-        end
-        sh "chmod 0600 #{path.first}/credentials"
-        sh "gem push pkg/#{gem}"
+        version = `bundle exec gem list`
```

Comment:
We know what `ENV["PACKAGE"]` is, presumably because it is set when starting the release build. We know what the version is from the release PR. Why don't we have `ENV["VERSION"]` here that is populated when the release build is started and to make sure that the version matches. Grabbing the version from the gem list negates the benefit of having the check, doesn't it?

Reply:
I agree that having `ENV["VERSION"]` set by kokoro is a better flow. It isn't currently and I believe would require some adjustments to autorelease/releasetool to make it work. I will try to get things working that way in the long run. For now, this at least makes it unnecessary to publish docs manually for each release.